src/blog/lagom-1-5-0-M1.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Lagom 1.5.0-M1 released!
+date: 2018-07-25
+author_github: ignasi35
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.5.0-M1
+---
+
+We are happy to announce the availability of Lagom 1.5.0-M1. This is the first milestone towards the releasing of Lagom 1.5.0 which moves Lagom closer to Play and Akka. This release includes Play 2.7.0-M2 and starts moving towards supporting scala 2.13.
```

Comment:
I feel like many readers won't understand what "moves Lagom closer to Play and Akka" means, even with the paragraph below. I think highlighting that it includes Play 2.7.0-M2 is good, since that's very specific. We could also mention the update to Akka HTTP 10.1 in the server implementation.

I think we can also say that we are laying the ground work for integration with Akka gRPC, and that Akka gRPC is being developed as a cross-product feature that can be used in services built with Akka, Lagom, or Play, and can be used to communicate between them. Then, the next paragraph could start with something like: "This cross-product collaboration is the first of many more to come. The motivation behind developing gRPC support as a cross-product feature is twofold: Lagom, Play and Akka users will be able to move between them more easily…" (continuing with the rest of what you wrote). I think something like that would add more context for Lagom users who haven't really seen behind the scenes.

"Starts moving towards supporting Scala 2.13" is a bit ambiguous as well. We haven't really done anything concrete to support Scala 2.13 in M1 as far as I can see, and it's not guaranteed that we will support Scala 2.13 in 1.5.0. It's going to depend on the timing of some other dependencies and could easily slip to a future version, so I'd rather not include it in announcements until we have more certainty.

I'd also like to emphasize up front that this is an unsupported pre-release, and that we don't guarantee API stability or binary compatibility between milestones, or from milestones to the final release. Maybe we can take some language from the [Play 2.7.0-M1 release announcement](https://blog.playframework.com/play-2-7-0-milestone-1-released/):

> Like all milestone releases, the primary goal is to get feedback, so please let us know if something isn't working or you see something that should be improved.

> Please note that these all _[sic]_ intended to be a preview of the 2.7.0 release and we would not recommend using in a production application. We have made an effort to introduce most major API changes in this release, but we reserve the right to break everything in a future milestone or release candidate. The good news is that there’s still a chance to provide feedback, so please try it out and let us know what you think. Don’t hesitate file issues on the project or start a discussion on our forums if you have suggestions.